spec/mailers/nsm/feedback_messages/further_information_request_feeback_spec.rb

```diff
@@ -12,7 +12,7 @@
   let(:ufn) { '123456/001' }
   let(:main_defendant_name) { 'Tracy Linklater' }
   let(:defendant_reference) { 'MAAT ID number: AB12123' }
-  let(:claim_total) { '£325.97' }
```

Comment:
Why did this value change?

Reply:
In most of our tests our "pricing" values in the blob were unrealistic numbers, and we were using them to calculate the totals. Now we're not using values in the blob but rather actual pricing data from the gem, so the totals change.